spi/imx6ull-ecspi/libecspi.c

```diff
@@ -279,7 +293,7 @@ static void readFifo(int dev_no, uint8_t *in, size_t len)
 		*in++ = (word >> 24) & 0xFF;
 		*in++ = (word >> 16) & 0xFF;
 		*in++ = (word >> 8) & 0xFF;
-		*in++ = (word) & 0xFF;
+		*in++ = (word)&0xFF;
```

Comment:
Why change that? Was ok. clang format might be confused because of redundant parenthesis
```suggestion
		*in++ = word & 0xFF;
```